build/configs/artik05x/artik05x_download.sh

```diff
@@ -79,6 +80,7 @@ Options:
 	BL2                           write Secondary Bootloader image into FLASH
 	SSSFW                         write Secure Element Firmware image into FLASH
 	WLANFW                        write WiFi Firmware image into FLASH
+        USERFS                        write user FS to FLASH(SMARTFS)
```

Comment:
Could you check all of indentation you modified?

Reply:
ok.Done.